api/pom.xml

```diff
@@ -21,7 +21,7 @@
     <parent>
         <groupId>com.okta.sdk</groupId>
         <artifactId>okta-sdk-root</artifactId>
-        <version>3.0.4-SNAPSHOT</version>
+        <version>4.0.0-SNAPSHOT</version>
```

Comment:
We should set the version to `3.1.0-SNAPSHOT` instead (`minor` version updgrade)